client/lib/feed-stream-store/feed-stream.js

```diff
@@ -237,7 +234,7 @@ export default class FeedStream {
 		do {
 			const key = this.postKeys[ i ];
 			if ( ! key.isGap ) {
-				date = FeedPostStore.get( key )[ this.dateProperty ];
+				date = key[ this.dateProperty ];
```

Comment:
Just to satisfy my curiosity: why do we no longer need `FeedPostStore` involved here?